about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Capstone Project 1</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+</head>
+
+<body>
+	<header class="container">
+		<nav class="sub-container1">
+			<ul class="link1">
+				<li class="list-item"><a href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="list-item"><a href="#"><i class="fa fa-twitter"></i></a></li>
+				<li class="list-item"><a href="#">English</a></li>
+				<li class="list-item"><a href="#">My Page</a></li>
+				<li class="list-item"><a href="#">Logout</a></li>
+			</ul>
+		</nav>
+		<nav class="sub-container2">
+			<div class="logo">
+				<a href="index.html"><img src="./images/logo2.png" alt=""></a>
+			</div>
+			<ul class="link2">
+				<li class="list-item2 mobile"><a href="index.html">Home</a></li>
+				<li id="red"><a href="index.html">About</a></li>
+				<li class="list-item2"><a href="#">Program</a></li>
```

Comment:
- [ ] Please as mentioned by the previous reviewer, kindly consider displaying `home` in place of about.
NB: the navigation is working well. when we click on the about link in the about page it navigates to the home page. all you have to do is change the text